core/field.js

```diff
@@ -196,20 +223,17 @@ Blockly.Field.prototype.EDITABLE = true;
 Blockly.Field.prototype.SERIALIZABLE = false;
 
 /**
- * Configure the field based on the given map of options.
- * @param {Object} opt_config The map of options to configure the field
- *    based on.
- * @private
+ * Process the configuration map passed to the field.
+ * @param {Object} config A map of options used to configure the field. See
```

Comment:
Looks like config can't be null.  {!Object}

Reply:
Yep